schematic/utils/csv_utils.py

```diff
@@ -366,8 +366,8 @@ def create_schema_classes(schema_extension: pd.DataFrame, se: SchemaExplorer) ->
                     # this attribute is a property, set the parent to the domain class of this attribute
                         parent = se.get_class_by_property(attribute["Attribute"])
                         if not parent:
-                            print("ERROR: Listed valid value " + val + " for attribute " + attribute["Attribute"] + " must have a class parent!")
-                            print("Could not add extension to schema!")
+                            logger.error(f"""Listed valid value: {val}, for attribute: {attribute["Attribute"]} """
+                                        "must have a class parent. The extension could not be added to the schema.")
                             exit()
```

Comment:
We should remove all instances of `exit()` and replace with specific exceptions.

Reply:
+1

Reply:
`ValueError` made the most sense for all the `exit()` statements, so I replaced all occurrences of `exit()` with `raise ValueError()`.